modules/ROOT/pages/http/http-authentication.adoc

```diff
@@ -2,357 +2,496 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: http, authentication, security, users, connectors, anypoint, studio, oauth, basic auth, digest
 
-Use the authentication that you configure in HTTP requests when your Mule app is sending requests to a service that requires authentication, such as the Github OAuth2 server described in <<OAuth2 - Authorization Code>>. In this case, your Mule app is the client. On the other hand, when you want to protect your API or app from _receiving requests_ from unauthorized access, use an API Manager policy, such as the xref:api-manager::external-oauth-2.0-token-validation-policy.adoc[OAuth 2.0 Access Token Enforcement Using External Provider] policy.
+Use the authentication that you configure in HTTP requests when your Mule runtime (Mule) app is sending requests to a service that requires authentication, such as the GitHub OAuth2 server described in <<OAuth2 - Authorization Code>>. In this case, your Mule app is the client. On the other hand, when you want to protect your API or app from receiving requests from unauthorized access, use an API Manager policy, such as the xref:api-manager::external-oauth-2.0-token-validation-policy.adoc[OAuth 2.0 Access Token Enforcement Using External Provider] policy.
```

Comment:
`app is sending requests to a service that requires authentication` -> `app uses the <HTTP Connector> to send requests to a service that requires authentication`